src/utils/tabs.js

```diff
@@ -50,10 +50,15 @@ export function getSourceAnnotation(
 ) {
   const sourceId = source.get("id");
   const sourceMetaData = getMetaData(sourceId);
+  const whichFramework =
+    sourceMetaData && sourceMetaData.whichFrameworkComponent
```

Comment:
i think `sourceMetaData.framework` would be simpler here... we can always add more specificity when we need it.

Reply:
Sure, will be much shorter.